packages/sdk-router/src/sdk.test.ts

```diff
@@ -409,6 +425,36 @@ describe('SynapseSDK', () => {
       expect(data?.length).toBeGreaterThan(0)
       expect(to?.length).toBeGreaterThan(0)
     })
+    it('throws if origin == destination', async () => {
+      const chainIds = [1, 42161]
+      const providers = [ethProvider, arbitrumProvider]
+      const Synapse = new SynapseSDK(chainIds, providers)
+      const result = await Synapse.bridgeQuote(
+        1,
+        42161,
+        '',
+        '',
+        BigNumber.from('1000000000000000000')
+      )
+      if (!result) {
+        throw Error
+      }
+      const { originQuery, destQuery } = result
+      await expect(
+        Synapse.bridge(
+          '0x0AF91FA049A7e1894F480bFE5bBa20142C6c29a9',
+          '0x7e7a0e201fd38d3adaa9523da6c109a07118c96a',
+          1,
+          1,
+          '',
+          BigNumber.from('10000000000000000000'),
+          originQuery!,
+          destQuery!
+        )
+      ).rejects.toThrowError(
+        'Origin chainId cannot be equal to destination chainId'
+      )
+    })
```

Comment:
This looks good. Think if we wanted to refactor repeated uses of a successful bridgeQuote down the line, we could use `beforeEach` and a `setupTest` helper

```
describe('bridge', () => {
  let Synapse
  let result
  let originQuery
  let destQuery

  const setupTest = async (fromChainId, toChainId, tokenIn, tokenOut, amountIn, chainIds, providers) => {
    Synapse = new SynapseSDK(chainIds, providers);
    result = await Synapse.bridgeQuote(
      fromChainId,
      toChainId,
      tokenIn,
      tokenOut,
      amountIn
    );
    if (!result) {
      throw Error
    }
    originQuery = result.originQuery;
    destQuery = result.destQuery;
  }

  describe('valid chain ids', () => {
    beforeEach(async () => {
      await setupTest(testVars)
    });

    it('successful bridge', async () => {
     const { data, to } = await Synapse.bridge(
       ...vars,
       originQuery!,
       destQuery!
     )
     expect(data?.length).toBeGreaterThan(0)
     expect(to?.length).toBeGreaterThan(0)
    })
  })

  describe('invalid chain ids', () => {
    beforeEach(async () => {
      await setupTest(testVars)
    });

    it('chain ids are the same, throws error', async () => {
      await expect(Synapse.bridge(...vars, originQuery!, destQuery!)).rejects.toThrowError('Origin chainId cannot be equal to destination chainId')
    })
  
    it('chain ids we dont support', async () => {
      // throws different error
    })
  })
})
```